openmetadata-service/src/main/java/org/openmetadata/service/elasticsearch/ElasticSearchIndexDefinition.java

```diff
@@ -272,7 +272,8 @@ class ParseTags {
     if (!tags.isEmpty()) {
       List<TagLabel> tagsList = new ArrayList<>(tags);
       for (TagLabel tag : tagsList) {
-        if (tag.getTagFQN().toLowerCase().matches("(.*)tier(.*)")) {
+        String tier = tag.getTagFQN().split("\\.")[0];
+        if (tier.equalsIgnoreCase("tier")) {
```

Comment:
@mohityadav766 let's use `FullyQualifiedName.getRoot()` method for this. That way we are using a method that is tested to get the root of the FQN instead of doing split in different parts of the code. All FQN processing should happen with that util class.